doc/custom-metadata-feature.md

```diff
@@ -4,26 +4,28 @@ Custom metadata can be applied and retrieved for each camera added to the servic
 
 ## Usage
 
-- The `CustomMetadata` map is an element in the `ProtocolProperties` device field. It is initialized to be empty on discovery, so the user can add their desired fields. Otherwise, the user can pre-define this field in a camera.toml file.
+- The `CustomMetadata` map is an element in the `ProtocolProperties` device field. It is initialized to be empty on discovery, so the user can add their desired fields. Otherwise, the user can pre-define this field in a camera.yaml file.
 
 ### Preset Custom Metadata
 
-If you add pre-defined devices, set up the `CustomMetadata` object as shown in the [camera.toml.example file](../cmd/res/devices/camera.toml.example).
-
-```toml
-# Pre-defined Devices
-[[DeviceList]]
-Name = "Camera001"
-ProfileName = "onvif-camera"
-Description = "onvif conformant camera"
-  [DeviceList.Protocols]
-    ... 
-    [DeviceList.Protocols.CustomMetadata]
-    Location = "Front door"
-    Color = "Black and white"
+If you add pre-defined devices, set up the `CustomMetadata` object as shown in the [camera.yaml.example file](../cmd/res/devices/camera.yaml.example).
+
+```yaml
+DeviceList:
+  - Name: Camera001
+    ProfileName: onvif-camera
+    Description: onvif conformant camera
+    Protocols:
+      Onvif:
+        Address: 192.168.12.123
+        Port: '80'
+        FriendlyName: Home camera
+        MACAddress: 'aa:bb:cc:dd:ee:ff'
+      CustomMetadata:
+        Location: Front door
+        Color: Black and white
 ```
```

Comment:
The point of this example is to focus on CustomMetadta. Please use `...` in place of the `Onvif` section so it matches what was in the TOML example